packages/angular_devkit/build_angular/src/builders/browser-esbuild/index.ts

```diff
@@ -595,6 +597,28 @@ export async function* buildEsbuildBrowser(
     outputFiles?: OutputFile[];
     assetFiles?: { source: string; destination: string }[];
   }
+> {
+  yield* buildEsbuildBrowserInternal(userOptions, context, infrastructureSettings);
```

Comment:
nit: Since this function is only passing through the internal one, it could just be a normal function that directly returns the iterator created by `buildEsbuildBrowserInternal`.

Reply:
Good point, simplified to `return buildEsbuildBrowserInternal(/* ... */);`.